gensim/test/test_doc2vec.py

```diff
@@ -306,6 +306,16 @@ def testTrainWarning(self, l):
                 model.alpha += 0.05
         warning = "Effective 'alpha' higher than previous training cycles"
         self.assertTrue(warning in str(l))
+
+    @log_capture()
+    def testLoadOnClassWarning(self):
+        """Test if logger warning is raised when loading doc2vec model on instance"""
+        model = doc2vec.Doc2Vec(DocsLeeCorpus(), min_count=1)
+        model.save(testfile())
+        load_on_instance = doc2vec.Doc2Vec()
+        model = load_on_instance.load(testfile())
+        warning = "Load methods should be called on a class object."
+        self.assertTrue(warning in str(l))
```

Comment:
No of course it wouldn't work. Sorry, I was trying something you weren't supposed to see this.